code/modules/reagents/reagent_containers/chem_bag.dm

```diff
@@ -0,0 +1,70 @@
+/obj/item/reagent_containers/chem_bag
+	name = "chemical bag"
+	desc = "Contains chemicals used for transfusion. Must be attached to an IV drip."
+	icon = 'icons/obj/bloodpack.dmi'
+	icon_state = "bloodpack"
+	volume = 200
+	fill_icon_thresholds = list(10, 20, 30, 40, 50, 60, 70, 80, 90, 100)
+	reagent_flags = TRANSPARENT | ABSOLUTELY_GRINDABLE
+	var/label_name // this is to support when you don't want to display "chemical bag" part with a custom name
+
+/obj/item/reagent_containers/chem_bag/Initialize(mapload)
+	. = ..()
+	if(!icon_state)
+		icon_state = "bloodpack"
+		update_icon()
+	if(label_name)
+		name = "[label_name] chemical bag"
+
+/obj/item/reagent_containers/chem_bag/examine(mob/user)
+	. = ..()
+	if(reagents)
+		if(volume == reagents.total_volume)
+			. += "<span class='notice'>It is fully filled.</span>"
+		else if(!reagents.total_volume)
+			. += "<span class='notice'>It's empty.</span>"
+		else
+			. += "<span class='notice'>It seems [round(reagents.total_volume/volume*100)]% filled.</span>"
+
+/obj/item/reagent_containers/chem_bag/epinephrine
+	label_name = "epinephrine"
```

Comment:
This unusued presets are basically from my own idea of this:
Remove sleeper, and medbay now starts with these 200u bags of basic chems. IV drip UI is reworked that you can adjust injection level(as TGUI)
80u bags are presets for Bacon's sleeper rework, so that you need to print more from the chemistry.